### DISCUSSION TITLE
How to build spdlog-rs on IOS

### DISCUSSION THREAD

**Discussion by newbeeAirfeeen** [SpriteOvO/spdlog-rs#35] [Q&A]
I have written an ios sdk now. This sdk uses spdlog-rs. But when I build the sdk of the ios general library. The following command was used:
`cargo lipo`
The compiler told me that there is no corresponding function on IOS。

```
spdlog-rs-0.3.10/src/record.rs:291:57
    |
291 |     TID.with(|tid| *tid.borrow_mut().get_or_insert_with(get_current_tid_inner))
    |                                                         ^^^^^^^^^^^^^^^^^^^^^ not found in this scope
```

I downloaded the source code and checked the relevant calls in libc, and found that there seems to be no corresponding function. If I want to build on ios, is it not supported by the current version yet.
Is it possible to change the way of thinking and use other methods to obtain the thread id when building on ios

**Comment by SpriteOvO** [comment]
I guess iOS should be easily supported. I have created issue #36, let's track the progress over there.
